tap_linkedin_ads/client.py

```diff
@@ -160,19 +161,41 @@ def __enter__(self):
     def __exit__(self, exception_type, exception_value, traceback):
         self.__session.close()
 
+    @backoff.on_exception(backoff.expo,
+                          (requests.exceptions.ConnectionError, requests.exceptions.Timeout),
+                          max_tries=5,
+                          factor=2)
+    def is_token_expired(self):
+        """
+        Function to check if the access token is expired.
+        """
+        current_time = time.time()
+        headers = {
+            "Content-Type": "application/x-www-form-urlencoded"
+        }
+        payload = 'client_id={}&client_secret={}&token={}'.format(self.__client_id, self.__client_secret, self.__access_token)
+        response = self.__session.post("https://www.linkedin.com/oauth/v2/introspectToken", headers=headers, data=payload)
+
+        if response.status_code != 200:
+            raise_for_error(response)
+
+        # Subtracting 2 days from the expiration date to avoid the failure of the token in case of a longer sync run.
+        return response.json()['expires_at'] - 172800 < current_time
```

Comment:
`return response.json()['expires_at'] - 172800 < time.time()`

can we change the line like above. So, we get accurate time

Reply:
Replaced `current_time` with `time.time()`